src/styles/app-text-input/AppTextInput.scss

```
@@ -67,6 +67,11 @@
     box-sizing: border-box;
     width: 100%;
 
+    &:focus-visible {
+      outline: none;
+      border: 2px solid #000;
```

Use color from design system
```suggestion
      border: 2px solid $color-primary;
```